config/packages/Centreon.yaml

```diff
@@ -111,6 +111,7 @@ services:
 
     Centreon\Domain\MonitoringServer\Interfaces\MonitoringServerConfigurationRepositoryInterface:
         class: Centreon\Infrastructure\MonitoringServer\Repository\MonitoringServerConfigurationRepositoryApi
+        public: true
```

Comment:
```suggestion
```

Reply:
If I remove this line, I have this error:
`The "Centreon\Domain\MonitoringServer\Interfaces\MonitoringServerConfigurationRepositoryInterface" service is private, you cannot replace it.`